training/src/components/Button/index.js

```diff
@@ -0,0 +1,3 @@
+import ButtonField from './Button';
```

Comment:
instead of this you can simple use
`
export { default as ButtonField } from './Button';`